static/golang/sample-source.html

```diff
@@ -0,0 +1,4 @@
+<html><head>
+    <meta name="go-import" content="knative.dev/sample-source git https://github.com/knative-sandbox/sample-source">
```

Comment:
hmm. don't you need to delete the old record?

Reply:
nobody ever created it.

Reply:
ack

Reply:
Actually, wondering how `dot-dev` worked for it before: https://github.com/knative/test-infra/pull/2244/files